libcloudforensics/gcp.py

```diff
@@ -91,6 +91,134 @@ def CreateService(service_name, api_version):
   return service
 
 
+class GoogleCloudLog:
+  """Class representing a Google Cloud Logs.
+
+  Attributes:
+    project_id: Project name.
+    gcl_api_client: Client to interact with GCP logging API.
+
+  Example use:
+    # pylint: disable=line-too-long
+    gcp = GoogleCloudLog(project_id='your_project_name')
+    gcp.ListLogs()
+    gcp.ExecuteQuery(filter='resource.type="gce_instance" labels."compute.googleapis.com/resource_name"="instance-1"')
+    See https://cloud.google.com/logging/docs/view/advanced-queries for filter details.
+  """
+
+  LOGGING_API_VERSION = 'v2'
+
+  def __init__(self, project_id):
+    """Initialize the GoogleCloudProject object.
+
+    Args:
+      project_id (str): The name of the project.
+    """
+
+    self.project_id = 'projects/' + project_id
```

Comment:
Could we keep this as just `project_id` so it stays consistent with the `project_id` attribute that's defined in the other GoogleCloud* classes?